exercises/dominoes/dominoes.spec.js

```diff
@@ -0,0 +1,129 @@
+// import { chain } from './dominoes'
+import { chain } from './dominoes'
+
+function runTest(dominoes, expected) {
+  const result = chain(dominoes)
+
+  // 1. The submitted code claims there is a chain if and only if there actually is one.
+  if (expected) {
+    it('Should have a chain', () => {
+      expect(result).not.toBe(null)
+    })
+  } else {
+    it('Should not have a chain', () => {
+      expect(result).toBe(null)
+    })
+  }
+
+  if (!expected) return
```

Comment:
This is not required, `expect` will raise error and rest of this function will not be executed.

Reply:
But the rest of the tests are irrelevant if `result` is expected to be `null`

Reply:
Instead, create two helper functions:
```
runTestsExpectingChain
runTestsExpectingNull
```

or similar.

It will remove the rather cryptic boolean value, and make it clear that the tests for "a chain" are different than "no chain".